xla/stream_executor/cuda/cuda_blas_lt.cc

```diff
@@ -47,8 +50,10 @@ limitations under the License.
     return std::move(value);                                              \
   }()
 
-namespace stream_executor {
-namespace cuda {
+namespace stream_executor::cuda {
```

Comment:
I think in general we prefer the separate lines for nested namespace declarations